orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/SaltOrchestrator.java

```diff
@@ -163,6 +163,9 @@ public class SaltOrchestrator implements HostOrchestrator {
     @Value("${cb.max.salt.database.dr.retry:300}")
     private int maxDatabaseDrRetry;
 
+    @Value("${cb.max.salt.database.dr.retry:600}")
```

Comment:
same property name reused, please use some other name

Reply:
Updated it to be `cb.max.salt.database.dr.retry.long:600` Thank you!!!